dozer-integrations/dozer-spring-support/dozer-spring4/src/main/resources/META-INF/spring.schemas

```diff
@@ -0,0 +1 @@
+http\://dozermapper.github.io/schema/dozer-spring.xsd=../../../../../../schema/src/main/resources/schema/dozer-spring.xsd
```

Comment:
is this correct? shouldn't it be path within the classpath

Reply:
No. This is not correct. Updated.